src/commands/indexer/index.ts

```diff
@@ -944,7 +966,8 @@ export default class Indexer extends HealthCheck {
 
         case SqsEventName.AvailableOperatorJob: {
           try {
-            const crossChainMessageType = this.getCrossChainMessageType(interestingEvent.eventName)
+            // TODO: it's always Unknown. This field can be safely removed in the future, as it is not utilized by the processor.
```

Comment:
What is always unknown?

Reply:
The CrossChainMessageType associated with the AvailableOperatorJob is always Unknown. Unfortunately, there's no straightforward method to verify the CrossChainMessageType using the transaction logs. To address this, our processor has a function that calls the API to retrieve this information.